docs/loom-testing.md

```diff
@@ -0,0 +1,32 @@
+# Loom Testing
```

Comment:
`docs/` are for Python interface document. This will not be compiled and displayed.